agent/src/containeroptions/dynamiccontaineroptions.cpp

```diff
@@ -40,4 +40,24 @@ bool DynamicContainerOptions::enableWriteBuffer() const
     return m_enableWriteBuffer;
 }
 
+void DynamicContainerOptions::setEnableTemporaryFileSystemWriteBuffers(bool enabled)
+{
+    m_enableTemporaryFileSystemWriteBuffers = enabled;
+}
+
+bool DynamicContainerOptions::enableTemporaryFileSystemWriteBuffers() const
```

Comment:
I don't know if we have any code guidelines for this, but I am used to that getters for bools are "isVariable" and getters for other types are "getVariable". Not sure if we have had or should have that in SC.